civis/cli/__main__.py

```diff
@@ -26,6 +27,11 @@
 from civis.cli._cli_commands import \
     civis_ascii_art, files_download_cmd, files_upload_cmd
 
+try:
+    FileNotFoundError
+except NameError:
+    FileNotFoundError = IOError
```

Comment:
This is probably worth a comment that `FileNotFoundError` isn't present in Python 2. What about defining our own `FileNotFoundError` class? `